draft-ietf-tls-esni.md

```diff
@@ -95,10 +95,11 @@ This document specifies a new TLS extension, called Encrypted Client Hello
 (ECH), that allows clients to encrypt their ClientHello to such a deployment.
 This protects the SNI and other potentially sensitive fields, such as the ALPN
 list {{?RFC7301}}. Co-located servers with consistent externally visible TLS
-configurations, including supported versions and cipher suites, form an
-anonymity set. Usage of this mechanism reveals that a client is connecting to a
-particular service provider, but does not reveal which server from the
-anonymity set terminates the connection.
+configurations and behavior, including supported versions and cipher suites and
+how they respond to incoming client connections, form an anonymity set. Usage
+of this mechanism reveals that a client is connecting to a particular service
+provider, but does not reveal which server from the anonymity set terminates
+the connection.
```

Comment:
I'm not sure what you mean by "how they respond to incoming client connections". Which certificate you send is arguably how part of "how they respond" but presumably we don't mean that! I suppose it could cover other externally-visible implementation choices like division of data into record-layer boundaries. That'd be a pretty solid example... but #519 talks about cipher suite selection, and that's already covered by configurations. So I'm just very confused right now.

Reply:
The idea here is that how a server responds, and how that response is visible to the attacker, influences the anonymity set. So on that basis I would consider what certificate they send to be covered here. Note that we don't need to be very rigorous here -- this is just the introduction and the goal is to convey the high-level goal of ECH.

Reply:
As above, I don't think we need to be rigorous or precise here. This suggestion is a lot more text than I think is warranted for this small point.

Reply:
@klinvill after thinking on this some more I decided to fold in a variant of your suggestion has a parenthetical. Please have another look?